### ISSUE TITLE
[BUG] Background not being replaced fully.

### ISSUE THREAD

**Issue by ItsRauf** [octref/polacode#125]
Issue: When taking code screenshots the container background is not fully replaced.
----
Steps to Reproduce:

1. Install Polacode
2. Start Polacode
3. Select code to take screenshot of
4. Observe

![polacode](https://user-images.githubusercontent.com/31735267/65726063-2acda000-e082-11e9-9624-43c3ce92c7e3.png)

----
Extension Info:
```
Name: Polacode
Id: pnp.polacode
Description: 📸  Polaroid for your code
Version: 0.3.3
Publisher: P & P
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=pnp.polacode
```

**Comment by octref** [comment]
Seems a regression by #121. I have reverted it and will make a new release.

@kufii fyi, I'll try to bring your changes in later.

**Comment by octref** [comment]
Should be fixed in 0.3.4.

**Comment by kufii** [comment]
@octref sorry about that, my next PR has that issue fixed, as well as the prior font family fix
